provider/openstack/client.go

```diff
@@ -201,10 +201,14 @@ func (c *ClientFactory) getClientState(options ...ClientOption) (client.Authenti
 		}
 
 		// If the AuthUserPassV3 client can authenticate, use it.
-		// Otherwise fallback to the v2 client.
+		// Otherwise, fall back to the v2 client.
```

Comment:
This comment isn't correct anymore, right?

Reply:
Good call -- moved to where it belongs.